fix: Remove limit on client POST payload size

riddell-stan opened the pull request:
Default `client_max_size` is 1 MiB. Model `data` is often greater. None
removes limit.

Closes #417
Closes #416

riddell-stan committed:
fix: Remove limit on client POST payload size

Default `client_max_size` is 1 MiB. Model `data` is often greater. None
removes limit.

Closes #417
Closes #416

ahartikainen reviewed (approved):
This is a good solution. If we ever go to "online" we can make it a bit more specific, but now we work only locally.

riddell-stan committed:
test: Pin pytest to 5.x

The pytest requirement should never have been removed.
Tests break in pytest 6.x.
